### PR TITLE
#144 - Code coverage improvements

### PR DESCRIPTION
**Pull request by marceloamadeu**
This PR is for #144 

- Added some tests to improve code coverage
- Minimum coverage changed

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#159](https://github.com/artipie/http/issues/159) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/0a87c6f6-ac40-4541-b30f-7ef5ca8c8ae4); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/artipie/http/rs/HeaderTest.java [hunk 1]
```diff
@@ -73,8 +74,19 @@ void shouldTrimValueLeadingWhitespaces(final String original, final String expec
         );
     }
 
+    @Test
+    void toStringHeader() throws Exception {
+        final String name = "name";
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/http/rs/HeaderTest.java:79]: @marceloamadeu let's inline this variable.

File: src/test/java/com/artipie/http/rs/HeaderTest.java [hunk 2]
```diff
@@ -73,8 +74,19 @@ void shouldTrimValueLeadingWhitespaces(final String original, final String expec
...
+        final String value = "value";
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/http/rs/HeaderTest.java:80]: @marceloamadeu let's inline this variable.

File: src/test/java/com/artipie/http/rs/RsWithStatusTest.java [hunk 3]
```diff
@@ -41,4 +43,15 @@ void usesStatus() throws Exception {
             new RsHasStatus(status)
         );
     }
+
+    @Test
+    void toStringRsWithStatus() throws Exception {
+        final RsStatus status = RsStatus.OK;
+        final Response origin = new RsWithStatus(RsStatus.OK);
```
Comment (carlosmiranda) [inline src/test/java/com/artipie/http/rs/RsWithStatusTest.java:50]: @marceloamadeu is nesting two `RsWithStatus` here really necessary for testing `toString` capability?

**Comment by marceloamadeu** [comment]
@carlosmiranda Resolved ... please check... Tks

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/artipie/http/issues/159#issuecomment-636442646))

@carlosmiranda This is what I know about this job in CT2E6TK9B, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#159](https://github.com/artipie/http/issues/159) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/CT2E6TK9B?q=%7Bjob%3A%27gh%3Aartipie%2Fhttp%23159%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Pull request merged**